shell/common/resource_cache_limit_calculator.h

```diff
@@ -0,0 +1,46 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_COMMON_RESOURCE_CACHE_LIMIT_CALCULATOR_
+#define FLUTTER_SHELL_COMMON_RESOURCE_CACHE_LIMIT_CALCULATOR_
+
+#include <cstdint>
+#include <unordered_map>
+
+#include "flutter/fml/macros.h"
+#include "flutter/fml/memory/weak_ptr.h"
+
+namespace flutter {
+class ResourceCacheLimitItem {
+ public:
+  // The expected GPU resource cache limit in bytes.
```

Comment:
Please document the threading expecations here (this will be called on the platform thread)

Reply:
done